content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,198 @@
+
+The replenishment report is a list of all products that have a negative forecast quantity.
+
+*Reordering rules* are used to ensure there's always a minimum amount of a product in-stock, in
+order to manufacture products and/or fulfill sales orders. When the stock level of a product reaches
+its minimum, Odoo automatically generates a purchase order with the quantity needed to reach the
+maximum stock level.
 
-Minimum Stock rules are used to ensure that you always have the minimum
-amount of a product in stock in order to manufacture your products
-and/or answer to your customer needs. When the stock level of a product
-reaches its minimum the system will automatically generate a procurement
-order with the quantity needed to reach the maximum stock level.
+Reordering rules can be created and managed in the replenishment report, or from the product form.
 
-Make to Order
+Make to order
 -------------
 
-The Make to Order function will trigger a Purchase Order of the amount
-of the Sales Order related to the product. The system will **not** check
-the current stock. This means that a draft purchase order will be
-generated regardless of the quantity on hand of the product.
+*Make to order (MTO)* is a procurement route that creates a draft purchase order (or manufacturing
+order) each time a sales order is confirmed, **regardless of the current stock level**.
+
+Unlike products replenished using reordering rules, Odoo automatically links the sales order to the
+|PO| or |MO| generated by the |MTO| route.
+
+Another difference between reordering rules and |MTO| is, with |MTO|, Odoo generates a draft |PO| or
+|MO| immediately after the |SO| is confirmed. With reordering rules, Odoo generates a draft |PO| or
+|MO| when the product's forecasted stock falls below the set minimum quantity.
+
+In addition, Odoo automatically adds quantities to the |PO| or |MO| as the forecast changes, so long
+as the |PO| or |MO| is not confirmed.
+
+The |MTO| route is the best replenishment strategy for products that are customized, and/or for
+products that have no stock kept on-hand.
+
 
 Configuration
 =============
 
-Minimum stock rules
--------------------
+Replenishment report and reordering rules
+-----------------------------------------
 
-The Minimum Stock Rules configuration is available through your
-Inventory module. In the Inventory Control menu select "Reordering Rule"
-in the drop down menu. There, click on "Create" to set minimum and
-maximum stock values for a given product.
+To access the replenishment report, go to :menuselection:`Inventory app --> Operations -->
+Replenishment.`
 
-.. image:: strategies/strategies01.png
+By default, the :guilabel:`Replenishment` report shows every product that needs to be manually
+reordered. If there is no specific rule for a product, Odoo assumes the :guilabel:`Min Quantity` and
+:guilabel:`Max Quantity` stock are both `0.00`
+
+.. image:: strategies/replenishment-report-dashboard.png
```

Comment:
This screenshot is very grainy and hard to read.
Can we re-take it in HD resolution, please? If needed, we can go up to 933px in width if 768px is too small.

Optional:
I prefer to put screenshots _after_ admonition blocks since they're only supplemental to the text. I think it's more valuable to have **note**, **tip**, **warning**, etc., in line with the instruction instead of buried after an image, like an after-thought, since they tend to convey important and sometimes timely/sensitive information.